pkg/jx/cmd/create_addon_cloudbees.go

```diff
@@ -195,6 +196,21 @@ To register to get your username/password to to: %s
 		return err
 	}
 
+	if o.Sso {
+		// wait for cert to be issued
+		certName := pki.CertSecretPrefix + "core"
+		log.Infof("Waiting for cert: %s...\n", util.ColorInfo(certName))
+		certMngrClient, err := o.CreateCertManagerClient()
+		if err != nil {
+			return errors.Wrap(err, "creating the cert-manager client")
+		}
+		err = pki.WaitCertificateExists(certMngrClient, certName, o.Namespace, 3*time.Minute)
+		if err != nil {
+			return err // this is already wrapped by the previous call
```

Comment:
Maybe provide a mindful message that the cert is not ready within the given timeout.

Reply:
It's already printing `Waiting for cert: tls-core...
error: waiting for certificate "tls-core" to be ready in namespace "jx".: timed out waiting for the condition`